spec/unit/document_spec.rb

```diff
@@ -88,7 +88,7 @@ class Post
     end
 
     it "should be recorded" do
-      Message.descendants.should == [Enter, Exit, Chat]
+      Message.descendants.should contain_exactly(Enter, Exit, Chat)
```

Comment:
Our own `direct_descendants` implementation changes the order of `descendants` from the previous implementation. I think it doesn't matter.

Reply:
LGTM, don't think the order is important.